Dockerfile

```diff
@@ -23,6 +23,7 @@ WORKDIR /explorer
 RUN apk add --no-cache ca-certificates docker
 COPY --from=backend_builder /tmp/gochain/* /usr/local/bin/
 COPY --from=frontend_builder /explorer/dist/* /explorer/
+COPY settings.json /explorer/
```

Comment:
We shouldn't bundle this file because it is mainnet specific, and won't apply to any other networks.

Reply:
Do you think in this case we should completely hide that link in UI?

Reply:
No, it's still useful statistics, we just will have bare addresses without name/url/region.

Reply:
Ya already figured out this, makes sense to me, need just to show it properly on frontend